queries/latex/highlights.scm

```diff
@@ -313,6 +298,15 @@
       (_) @markup.link.url))
   (#any-of? @_name "\\url" "\\href"))
 
+[
+  (line_comment)
+  (block_comment)
+  (comment_environment)
+] @comment @spell
+
+((line_comment) @keyword.directive
+  (#lua-match? @keyword.directive "^%% !TeX"))
```

Comment:
I'd say just put this to the end of the file. Who knows how many queries we will add in the future.